src/test/java/com/google/sps/data/DatastoreScheduledInterviewTest.java

```diff
@@ -184,4 +195,46 @@ public void failsGetScheduledInterview() {
     Optional<ScheduledInterview> actual = dao.get(2);
     Assert.assertEquals(Optional.empty(), actual);
   }
+
+  // Tests retrieving all scheduledInterviews for a particular user in a certain range
+  @Test
+  public void getsScheduledInterviewsInRange() {
+    dao.create(scheduledInterview1);
+    dao.create(scheduledInterview2);
+    dao.create(scheduledInterview3);
+    dao.create(scheduledInterview4);
+    List<ScheduledInterview> result =
+        dao.getScheduledInterviewsInRangeForUser(
+            "user@company.org",
+            Instant.parse("2020-07-06T19:00:10Z").toEpochMilli(),
+            Instant.parse("2020-07-06T21:00:10Z").toEpochMilli());
+    List<Entity> entities =
+        datastore
+            .prepare(new Query("ScheduledInterview").addSort("startTime", SortDirection.ASCENDING))
+            .asList(FetchOptions.Builder.withDefaults());
+    List<ScheduledInterview> scheduledInterviews = new ArrayList<ScheduledInterview>();
+    for (Entity entity : entities) {
+      scheduledInterviews.add(dao.entityToScheduledInterview(entity));
+    }
+    ScheduledInterview expectedScheduledInterview1 =
+        ScheduledInterview.create(
+            scheduledInterviews.get(1).id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
```

Comment:
Please get the instant objects from the original interview (i.e. `scheduledInterview1.when()...`). This will be clearer than hard-coding them again. The same goes for the interviewer and interviewee.

Reply:
Done